aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -467,6 +455,22 @@ When hosting a non-ASP.NET Core sub-app underneath an ASP.NET Core app, explicit
 
 ::: moniker-end
 
+Static asset links within the sub-app should use tilde-slash (`~/`) notation. Tilde-slash notation triggers a [Tag Helper](xref:mvc/views/tag-helpers/intro) to prepend the sub-app's pathbase to the rendered relative link. For a sub-app at `/subapp_path`, an image linked with `src="~/image.png"` is rendered as `src="/subapp_path/image.png"`. The root app's Static File Middleware doesn't process the static file request. The request is processed by the sub-app's Static File Middleware.
+
+If a static asset's `src` attribute is set to a relative URL (for example, `src="/image.png"`), the link is rendered without the sub-app's pathbase. The root app's Static File Middleware attempts to serve the asset from the root app's [webroot](xref:fundamentals/index#web-root-webroot), which results in a *404 - Not Found* response unless the static asset is available from the root app.
+
+To host an ASP.NET Core app as a sub-app under another ASP.NET Core app:
+
+1. Establish an app pool for the sub-app. Set the **.NET CLR Version** to **No Managed Code**.
+
+1. Add the root site in IIS Manager with the sub-app in a folder under the root site.
+
+1. Right-click the sub-app folder in IIS Manager and select **Convert to Application**.
+
+1. In the **Add Application** dialog, use the **Select** button for the **Application Pool** to assign the app pool that you created for the sub-app. Select **OK**.
+
+In some scenarios, URL path segments must be manipulated to make routing work properly. For more information, see <xref:host-and-deploy/proxy-load-balancer#deal-with-path-base-and-proxies-that-change-the-request-path>.
```

Comment:
The PathBase should not need to be altered manually in IIS scenarios, that's already built in.
https://github.com/aspnet/AspNetCore/blob/4d57528704d7ec54570f5960b3b9ddca503d5c89/src/IISIntegration/src/Microsoft.AspNetCore.Server.IIS/Core/IISServerSetupFilter.cs#L31